src/utils/dom.ts

```diff
@@ -35,3 +35,5 @@ export function getSlottedElements<T extends Element>(
     T
   >;
 }
+
+export const hydratedInvisibleClass = "hydrated--invisible";
```

Comment:
One last thing, maybe we can export a `CSS` object in case more host-level classes are needed.